### PR TITLE
Add presentations of BHNT#33 (closes #115)

### LINKED ISSUE #115: Need Google Docs poll info for BHNT#33
We have no information at all about BHNT#33. So we need all information from the Google Docs poll:

* Which projects where presented? By whom?
* Which one was the hack of the month?

### PR DESCRIPTION
**Pull request by vog**
Closes #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,8 +162,14 @@ This event was mislabeled \#46 on Meetup.
 
 2015-03-31 | 18:30 | [Meetup Event](https://www.meetup.com/Berlin-Hack-and-Tell/events/221394273/) | [Photos](https://www.meetup.com/Berlin-Hack-and-Tell/photos/26018542/) | [@c-base](https://www.c-base.org)
 
+* Tidus gem by Tobias
+* vimify by Philipp
+* YeahLetsDoThat Event Sourcing Demo by Uwe
```
Comment (ligi) [inline README.md]: add @uwekamper  - with https://github.com/uwekamper/yeahletsdothat

File: README.md [hunk 2]
```diff
@@ -162,8 +162,14 @@ This event was mislabeled \#46 on Meetup.
...
+* Android Color Thief by Henrique
+* Mailbox Collection Time Filter by Tobi
```
Comment (ligi) [inline README.md]: @johnjohndoe - that's you right? can you PR the links?

Comment (johnjohndoe) [inline README.md]: 👎 Not me, sry.

**Comment by ligi** [comment]
an I thought it is you with the mailbox times for OSM - AFAIR you have
talked about this at bhnt

**Pull request merged**